README.md

```diff
@@ -6,43 +6,29 @@
-visualization and analysis library targeted for GeoInformatics.
-
-![Historical hurricane data Visualization in GeoJS](http://opengeoscience.github.io/geojs/examples/hurricanes/thumb.jpg)
-
-Documentation
-=============
-You can find more information on GeoJS installation and usage in our [Documentation](http://geojs.readthedocs.org/en/latest/index.html).
-For live examples visit [Github.io](http://opengeoscience.github.io/geojs/examples/index.html).
-
-Support
-=======
-We would love to get your feedback and bug reports. Please send us an email at
-[geojs-developers](http://public.kitware.com/mailman/listinfo/geojs-developers)
-for questions related to feature requests, roadmap or in-depth
-technical discussions. Use [geojs-users](http://public.kitware.com/mailman/listinfo/geojs-users)
-if you have general questions or bug reports on GeoJS.
-
-License
-=======
-Copyright 2015 Kitware Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-
-Get Involved
-============
-GeoJS is developed to meet needs of developers and domain-specific researchers,
-and we are always open for collaboration. We have been doing open source development
-for the last 17 year and understand and appreciate what it takes to develop open
-source software. So fork GeoJS, do great things and let us know if we can help.
+[Epidemico](http://epidemico.com/). GeoJS is intended to bridge the gap between
+[GIS](https://en.wikipedia.org/wiki/Geographic_information_system),
+[Geospatial Visualization](https://en.wikipedia.org/wiki/Geovisualization),
+[Scientific Visualization](https://en.wikipedia.org/wiki/Scientific_visualization) and
+[Infovis](https://en.wikipedia.org/wiki/Information_visualization).
+GeoJS is not just a GIS library as users can create scientific plots
+such as vector and contour and can embed infovis plots using [D3.js](https://github.com/d3/d3).
+*GeoJS aims for high-performance visualization and interactive data exploration of scientific
+and geospatial location aware datasets*.
+
+GeoJS supports features such as *Point*, *Line*, *Polygon*, and advanced features
+such as [Pixelmap](http://opengeoscience.github.io/geojs/examples/pixelmap),
+[Contour](http://opengeoscience.github.io/geojs/examples/contour),
+[Heatmap](http://opengeoscience.github.io/geojs/examples/heatmap), and
+[Choropleth] http://opengeoscience.github.io/geojs/examples/choropleth).
+
+GeoJS can read and render data from [GeoJSON](http://opengeoscience.github.io/geojs/examples/geoJSON) file,
```

Comment:
Sounds good, I will push a commit to include both of these suggestions. I liked it.